src/entities/Budget/routes.ts

```diff
@@ -35,3 +36,7 @@ async function fetchBudgets(): Promise<TransparencyBudget[]> {
 async function getCurrentBudget(): Promise<CurrentBudget> {
   return await BudgetService.getCurrentBudget()
 }
+
+async function getExpectedAllocatedBudget(): Promise<Record<string, any>> {
```

Comment:
Wouldn't it be better to type what the endpoint returns instead of just 'any'?

Edit: the type is already defined, so it is simply `Promise<ExpectedBudget>`